app/components/Rewards.tsx

```diff
@@ -0,0 +1,105 @@
+import { EVENTS, REWARD_DATA, CERTIFICATIONS } from "@/constants/constants";
+import NFTDashboard from "../components/nft_events_rewards/NftEventsRewards";
+
+const MarketPlace: {
+  title: string;
+  desc: string;
+  price: number;
+  currency: string;
+  buttonText: string;
+}[] = [
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+];
+
+function Rewards() {
+  return (
+    <div className="flex flex-col gap-4 my-3 pr-6">
+      <p>Rewards</p>
+      <div className="bg-white flex flex-col gap-2 rounded-xl px-5 py-3 text-black font-semibold ">
+        <p>Token Balance</p>
+        <h2 className="text-4xl uppercase ">4,000 Wrld</h2>
+        <p className="text-xs font-light">≈ $125.00 USD</p>
+      </div>
+      {/* nft rewrad/ events */}
+      <NFTDashboard
```

Comment:
there is a component already created which matches what should be here instead of this component.

The nft_events_reward component is what you should import here.

Reply:
I will get on that.